src/less-style/inputs.less

```diff
@@ -78,20 +78,22 @@
 }
 
 .fd-commander {
-    display: flex;
     background-color: @cc-bg;
-    padding: 4px 1px 3px 3px;
-    margin-right: 1px;
+    padding: 5px;
 
     input {
         border: 1px solid @cc-neutral-hi;
-        flex: 1;
+        height: 19px;
+        width: calc(~"100% - 15px");
```

Comment:
no way! I didn't even know this was possible. Is that a Less thing, or is that supported in raw CSS? I guess some form of this must be supported in raw CSS since it's a dynamic calculation that can't be done at Less compile time... Awesome!